recipes/funcannot/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: funcannot
+  version: "v2.8"
+
+source:
+  url: https://github.com/BioTools-Tek/funcannot/archive/master.zip
+  md5: 2e70b503a1632998deba759b50c0c07f
+   
+
+build:
+  number: 0
+  #skip: True # [osx]
+
+requirements:
+  build:
+    - gcc
+    - zlib {{CONDA_ZLIB}}*
+    - qt
+
+  run:
+    - libgcc
+    - qt
```

Comment:
do you need to pin the qt version? version 4 or 5?

Reply:
It can build for both, but I'll lock it qt4 for now